packaging/bundle-validation/ci_run.sh

```diff
@@ -41,6 +41,16 @@ if [[ ${SPARK_RUNTIME} == 'spark2.4' ]]; then
   CONFLUENT_VERSION=5.5.12
   KAFKA_CONNECT_HDFS_VERSION=10.1.13
   IMAGE_TAG=flink1136hive239spark248
+elif [[ ${SPARK_RUNTIME} == 'spark3.0.2' ]]; then
+  HADOOP_VERSION=2.7.7
+  HIVE_VERSION=3.1.3
+  DERBY_VERSION=10.14.1.0
+  FLINK_VERSION=1.14.6
+  SPARK_VERSION=3.0.2
+  SPARK_HADOOP_VERSION=2.7
+  CONFLUENT_VERSION=5.5.12
+  KAFKA_CONNECT_HDFS_VERSION=10.1.13
+  IMAGE_TAG=flink1146hive313spark302
 elif [[ ${SPARK_RUNTIME} == 'spark3.1.3' ]]; then
```

Comment:
Does the tag `flink1146hive313spark302` needs to be prefabricated, seems no script really build this image.

Reply:
Please refer this PR
https://github.com/apache/hudi/pull/8767